includes/utils.php

```diff
@@ -195,10 +195,19 @@ function set_meta( $post_id, $meta ) {
 				$prev_value = maybe_unserialize( $existing_meta[ $meta_key ][ $meta_placement ] );
 			}
 
+			$is_json = false;
+
 			if ( ! is_array( $meta_value ) ) {
+				json_decode( $meta_value );
+				$is_json    = json_last_error() === 0 ? true : false;
 				$meta_value = maybe_unserialize( $meta_value );
 			}
 
+			// Adds slashes to a string or recursively adds slashes to strings within an array for the JSON
+			if ( function_exists( 'wp_slash' ) && ( $is_json || '_elementor_data' === $meta_key ) ) {
```

Comment:
I don't think we need the `function_exists` check here, as that function has been in WordPress since 3.6.0

Reply:
Also, wonder if we should introduce a new filter here that could be used to turn slashing on? I'm sure there's other types of data we aren't accounting for here that could use slashing and don't really want to have to manually support all those use cases (like we're supporting `_elementor_data` here). So adding a filter would allow others to hook in and turn this on as needed

Reply:
Or as [Peter mentions](https://github.com/10up/distributor/pull/888#issuecomment-1156007012), could just slash all meta which would simplify things here quite a bit

Reply:
Yeah, I'll update the PR this week :)

Reply:
> I don't think we need the `function_exists` check here, as that function has been in WordPress since 3.6.0

@dkotter I've removed the check but now PHPUnit says undefined function! here https://github.com/10up/distributor/runs/6985524501?check_suite_focus=true

Reply:
@sksaju To fix the tests, you'll need to mock the `wp_slash` function with an `\WP_Mock::userFunction( 'wp_slash', $args_array )` in the test suite. 

I've not used the mocking library yet but will try to figure out what the args array should be.

Reply:
I've added mocks for `wp_slash()` in 4f88fff9f9e35e866b302a1ce977146ad8aaeab3 which seems to fix the tests.

@dkotter Would you mind validating my use of the mocks?

Reply:
> I've added mocks for `wp_slash()` in [4f88fff](https://github.com/10up/distributor/commit/4f88fff9f9e35e866b302a1ce977146ad8aaeab3) which seems to fix the tests.
> 
> @dkotter Would you mind validating my use of the mocks?

This looks good. If wanted, you could set these up as [passthru functions](https://github.com/10up/wp_mock#passthru-functions), since in essence that's what we're doing here. But this is equivalent to that anyway.